modules/localgov_review_date/src/Plugin/Field/FieldWidget/ReviewDateWidget.php

```diff
@@ -62,7 +62,7 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
 
     // Get current review status object.
     $entity = $items->getEntity();
-    $langcode = $form_state->get('langcode');
+    $langcode = $form_state->get('langcode') ?? 'und';
```

Comment:
Can we change `und` to `LanguageInterface::LANGCODE_NOT_SPECIFIED`
You'll also need to add a `use Drupal\Core\Language\LanguageInterface;` statement at the top.